packages/vscode-extension/src/utils/projectStatusUtils.ts

```diff
@@ -16,6 +16,7 @@ export const RecordedActions: (keyof ProjectActionStatus)[] = [
   CommandKey.Deploy,
   CommandKey.Publish,
   CommandKey.OpenReadMe,
+  CommandKey.LocalDebug,
```

Comment:
is this key used? do not find any reference.

Reply:
![image](https://github.com/OfficeDev/TeamsFx/assets/55721214/09d53d78-f36a-4200-a19c-3bf99d9d8047)
This is used in office next steps to launch local debug for Office Add-ins

Reply:
Hi @GavinGu07, we cannot use the result of `fx-extension.localdebug` to adjust if debug succeed or fail, because this command just do `quickpick` and will always return success.

We use the below logic to do it separately. You can see in https://github.com/OfficeDev/TeamsFx/pull/11343/files#diff-c28d3097df56371fe766b0cc7e053472a9baefb8d565e7081db5ef80cfaf2c1f
![image](https://github.com/OfficeDev/TeamsFx/assets/15262146/c57feaed-f6cd-4c16-86dd-983ef8be0581)

Reply:
Thanks @Alive-Fish. @xuruiyao-msft is addressing this issue and she will give updates on it.

Reply:
Please help review the related PR: https://github.com/OfficeDev/TeamsFx/pull/11443